src/reference/config/zilla-cli.md

```diff
@@ -101,6 +102,14 @@ Examples:
 ./zilla help start
 ```
 
+### zilla version
+
+The `zilla version` command prints the version information of Zilla.
+
+```bash:no-line-numbers
+zilla version
+```
+
```

Comment:
````suggestion

```output:no-line-numbers
<Add expected output example>
```

````

Can you add an expected output example for what this command prints after it is run? you can see the [zilla tune](https://docs.aklivity.io/zilla/latest/reference/config/zilla-cli.html#zilla-tune) command in the docs for examples on what it looks like rendered.